docs/basic-features/image-optimization.md

```diff
@@ -117,7 +117,7 @@ module.exports = {
 
 The following Image Optimization cloud providers are included:
 
-- [Vercel](https://vercel.com): Works automatically when you deploy on Vercel, no configuration necessary. [Learn more](https://vercel.com/docs/next.js/image-optimization)
+- [Vercel](https://vercel.com): Works automatically when you deploy on Vercel, no configuration necessary. [Learn more](/docs/next.js/image-optimization)
```

Comment:
This needs to remain vercel.com because the relative path will be incorrect without it (remember this is deployed to nextjs.org)

Reply:
Ok, that makes sense. I was wondering why this one didn't match up with the others on the page